ballerina/main.bal

```diff
@@ -17,6 +17,8 @@
 const JSON_CONVERSION_ERROR = "FromJsonStringError";
 const CONVERSION_ERROR = "ConversionError";
 const ERROR_MESSAGE = "Error occurred while attempting to parse the response from the LLM as the expected type. Retrying and/or validating the prompt could fix the response.";
+const OBJECT_KEY = "result";
```

Comment:
OBJECT_KEY isn't self-explanatory. Either use RESULT or introduce a name that reflects the usage.